src/frontend/src/flows/authorize/postMessageInterface.ts

```diff
@@ -53,6 +53,7 @@ export type AuthResponse =
       kind: "authorize-client-success";
       delegations: Delegation[];
       userPublicKey: Uint8Array;
+      authnMethod: "pin" | "passkey" | "recovery";
```

Comment:
The type `AuthResponse` is not directly imported by the apps connecting to II, right?

It's just the type they receive. But adding a new mandatory field doesn't break anything.

Reply:
> not directly imported by the apps connecting to II

Can you clarify what you mean by imported?

There is currently no type library exposed by II, right. And it's just the return type, so this should be backwards compatible!

Reply:
That's exactly what I meant. Thanks for clarifying.